src/main/java/org/jabref/gui/WelcomePage.java

```diff
@@ -0,0 +1,209 @@
+    Label newLibraryLabel;
+    Label openLibrarylabel;
+    Label connectToDatabase;
+
+    Label importNewLib;
+
+    private static WelcomePage welcomeTabInstance = null;
+
+    public static WelcomePage getInstance(PreferencesService preferencesService,
+                                          StateManager stateManager,
+                                          FileUpdateMonitor fileUpdateMonitor,
+                                          TaskExecutor taskExecutor,
+                                          DialogService dialogService,
+                                          VBox welcomeBox) {
+        if (welcomeTabInstance == null) {
+            welcomeTabInstance = new WelcomePage(preferencesService, stateManager, fileUpdateMonitor, taskExecutor, dialogService, welcomeBox);
+        }
+        return welcomeTabInstance;
+    }
+
+    private WelcomePage(PreferencesService preferencesService,
+                        StateManager stateManager,
+                        FileUpdateMonitor fileUpdateMonitor,
+                        TaskExecutor taskExecutor,
+                        DialogService dialogService,
+                        VBox welcomeBox) {
+        this.preferencesService = preferencesService;
+        this.stateManager = stateManager;
+        this.fileUpdateMonitor = fileUpdateMonitor;
+        this.taskExecutor = taskExecutor;
+        this.dialogService = dialogService;
+        this.welcomeBox = welcomeBox;
+        welcomeBox.resize(50, 50);
+        initialiseWelcomePage();
+        runLabelTasks();
+    }
+
+    public void initialiseWelcomePage() {
+        welcomePageItems = new VBox();
+        // Need to set Padding for the Elements inside the Veritcal Box
+        welcomePageItems.setPadding(new Insets(10));
+        welcomePageItems.setAlignment(Pos.BASELINE_CENTER);
+        // Adding the title for the Welcome Tab
+        Label title = new Label();
+        title.setText("Welcome to JabRef – Your Bibliographic Management and Citation Tool");
+        title.setFont(Font.font("Arial", FontWeight.BOLD, 30));
+        welcomePageItems.getChildren().add(title);
+
+
+        addStartLabel();
+        items.setAlignment(Pos.CENTER);
+        welcomePageItems.getChildren().add(items);
+        welcomeBox.getChildren().add(welcomePageItems);
+    }
+
+    public void addStartLabel() {
+        Label start = new Label("Start:");
+        start.setStyle(headings);
+//        start.setFont(Font.font("Arial", FontWeight.BOLD, 30));
+
+        start.setTranslateX(start.getTranslateX() - 90);
+        items.getChildren().add(start);
+        items.setAlignment(Pos.CENTER_LEFT);
+        addNewLibraryButton();
+//        addOpenLibraryButton();
+        addConnectDataBaseButton();
+        addImportLibraryButton();
+    }
+
+    public void addNewLibraryButton() {
+        newLibraryLabel = new Label("New Library");
+        newLibraryLabel.setStyle(fontStyle);
+        newLibraryLabel.setOnMouseEntered(event -> {
```

Comment:
Controlsfx provides a Hypertext Link label. Already in use eg. in the about dialog